opensquirrel/instruction_library.py

```diff
@@ -1,16 +1,23 @@
+from __future__ import annotations
+
 from abc import ABC
+from collections.abc import Callable, Iterable, Mapping
+
+from opensquirrel.squirrel_ir import Gate, Measure
```

Comment:
Same here. Maybe GitHub diffs being tricky?

```suggestion
from opensquirrel.ir import Gate, Measure
```

Reply:
OK, I see they're gone now. Sorry about that!